test/index.test.js

```diff
@@ -21,6 +21,35 @@ describe('Integration Tests', () => {
     assert.equal(result.statusCode, 400);
   });
 
+  it('tests index with absolute run_query url', async () => {
+    const EXPECTED_HEADERS = {
+      'Cache-Control': 'max-age=600',
+      'Content-Type': 'application/json',
+    };
+    const { body, headers, statusCode } = await main({
+      __ow_path: '/https://adobeioruntime.net/api/v1/web/helix/helix-services/run-query@2.4.11/error500',
+    });
+
+    assert.ok(Array.isArray(body));
+    assert.deepEqual(EXPECTED_HEADERS, headers);
+    assert.equal(statusCode, 200);
+  }).timeout(6000);
+
+  it('tests index with relative run_query url', async () => {
```

Comment:
I'd copy (and adjust) one of the two tests into post-deploy, so that we can make sure it's working in Runtime, too.